api/openapi.yaml

```diff
@@ -1429,10 +1429,10 @@ paths:
           description: "problem deleting a demo"
 
   # Data Stores
-  /datastores:
+  /datastore:
```

Comment:
so if we add a plural form, it would leave the api almost unchanged, right? it's not hard to add that to the resource manager, do you think that would work?

Reply:
It makes sense. I'm finishing the first draft of the DataStore, and I'll add this functionality.

Reply:
Issue created: https://github.com/kubeshop/tracetest/issues/2387